.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,20 @@
+##
+ # Dependabot auto-merge config for merging patch and minor updates.
+ #
+ # Based on https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#enable-auto-merge-on-a-pull-request
+ # ______________________________________________________________________________
+ #
+ # Maintained by brightSPARK Labs.
+ # www.brightsparklabs.com
+ ##
+
+name: Dependabot auto-merge
```

Comment:
Rename this to to `Gradle Plugins Workflow`, and the file to `gradle-plugins.yml`.

This should be the only workflow file in the repo since it combines the logic from the previous workflows. Delete the other workflow files in `.github/workflows`.